frontend/src/metabase/admin/settings/components/SettingsPages/GeneralSettingsPage.tsx

```diff
@@ -0,0 +1,75 @@
+import { jt, t } from "ttag";
+
+import { useDocsUrl } from "metabase/common/hooks";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { Stack } from "metabase/ui";
+
+import { AdminSettingInput } from "../widgets/AdminSettingInput";
+import { AnonymousTrackingInput } from "../widgets/AnonymousTrackingInput";
+import { CustomHomepageDashboardSetting } from "../widgets/CustomHomepageDashboardSetting";
+import { HttpsOnlyWidget } from "../widgets/HttpsOnlyWidget";
+import { SiteUrlWidget } from "../widgets/SiteUrlWidget";
+
+export function GeneralSettingsPage() {
+  const { url: iframeDocsUrl } = useDocsUrl("configuring-metabase/settings", {
+    anchor: "allowed-domains-for-iframes-in-dashboards",
+  });
+
+  return (
+    <Stack gap="xl" maw="42rem" px="lg" py="sm">
+      <AdminSettingInput
+        name="site-name"
+        title={t`Site Name`}
+        description={t`The name used for this instance of Metabase.`}
+        inputType="text"
+      />
```

Comment:
Nit: is the spacing between components needed? I don't think we do it anywhere else? For me it just makes the file longer

Reply:
I don't really care, I think our philosophy is just no-quibbling-about-formatting, so if you think we should disallow blank lines between components, open an RFC for the frontend guild enforcing it in a lint rule or prettier config.